.github/workflows/label-cherry-pick.yaml

```diff
@@ -0,0 +1,71 @@
+name: Label Cherry Pick
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - closed
+
+jobs:
+  cherry_pick:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target_branch: ['23.lts.1+', '22.lts.1+', '21.lts.1+', '20.lts.1+', '19.lts.1+', '19.android.1+', 'rc_11', 'COBALT_9']
+    if: |
```

Comment:
Ideally there would be a way to determine `RUN_FOR_CURRENT_BRANCH` here instead of in the steps, but I'm not sure there is as there's no way to access `matrix.target_branch`.